### PR TITLE
@feature/next.js

### PR DESCRIPTION
**Pull request by ayodeji-ti**

### CODE REVIEW

**Comment by ayodeji-ti** [comment]
Next.js now handles the code sync, not express.js, hence there is only one server running, [loom video](https://www.loom.com/share/d0d160c439a946d9bc07987a4b85579d)

File: next.config.js [hunk 1]
```diff
@@ -1,45 +1,6 @@
...
-            onlyCompileBundledFiles: true,
-          },
-        },
-      ],
-    });
-
-    return config;
-  },
-}
+const nextConfig = {}
```
Comment (vinayak-sachdeva) [inline next.config.js:4]: don't we need this config now?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trilogy-group/tu2k22-openchakra",
-  "version": "1.0.0",
+  "version": "1.0.6",
```
Comment (vinayak-sachdeva) [inline package.json:3]: version it 0.0.x until first release

File: src/components/CodePanel.tsx [hunk 3]
```diff
@@ -5,15 +5,24 @@ import { generateCode } from '~utils/code'
 import theme from 'prism-react-renderer/themes/nightOwl'
 import { useSelector } from 'react-redux'
 import { getComponents } from '~core/selectors/components'
+import axios from 'axios'
 
 const CodePanel = () => {
   const components = useSelector(getComponents)
   const [code, setCode] = useState<string | undefined>(undefined)
+  const API_ENDPOINT = axios.create({
+    baseURL: `${window.location.href}api`,
```
Comment (vinayak-sachdeva) [inline src/components/CodePanel.tsx:14]: Inelegant, what if it doesn't end with '/', please find some alternate solution

Comment (vinayak-sachdeva) [reply src/components/CodePanel.tsx:14]: can you try localhost:3003

**Review by vinayak-sachdeva — CHANGES_REQUESTED** [review]
Please address all the comments

File: package.json (hunk 2 above)
Comment (ayodeji-ti) [reply package.json:3]: okay

**Pull request merged**